waspc/src/Wasp/Generator/ServerGenerator/Common.hs

```diff
@@ -99,3 +102,14 @@ dotEnvServer = [relfile|.env.server|]
 -- client SDK identifiers. Useful when creating `context.entities` JS objects in Wasp templates.
 entityNameToPrismaIdentifier :: String -> String
 entityNameToPrismaIdentifier entityName = toLower (head entityName) : tail entityName
+
+buildEntityData :: String -> Aeson.Value
```

Comment:
This is doing something quite specific while having a quite generic name.
So I would either give it more specific name, or inline it.

For example name could be: makeJsonWithEntityNameAndPrismaIdentifier .

Reply:
I see now that it existed before already, but still, I would use this opportunity to improve this. Before it was localized so that is why it was ok -> now it is in Common and used in multiple places so we need a better name.

Reply:
I'll change this in the my third PR (#962) to minimize conflicts.